Rebus.Idempotency.MySql/MySqlMessageStorage.cs

```diff
@@ -42,16 +42,16 @@ public async Task<MessageData> Find(MessageId messageId)
                             WHERE s.`message_id` = @message_id and s.`defer_count`= @defer_count
                         ";
                     command.Parameters.Add(command.CreateParameter("message_id", DbType.String, messageId.OriginalMessageId));
-                    command.Parameters.Add(command.CreateParameter("defer_count", DbType.Int32, messageId.DeferCount));
+                    command.Parameters.Add(command.CreateParameter("defer_count", DbType.SByte, messageId.DeferCount));
```

Comment:
Doesn't this mean `signed`? This should be `unsigned` and I believe MySQL supports that...